scss/_variables.scss

```diff
@@ -1028,7 +1028,7 @@ $accordion-button-padding-y:              $accordion-padding-y !default;
 $accordion-button-padding-x:              $accordion-padding-x !default;
 $accordion-button-color:                  $accordion-color !default;
 $accordion-button-bg:                     $accordion-bg !default;
-$accordion-transition:                    $btn-transition, border-radius .15s ease !default;
+$accordion-transition:                    $btn-transition, border-radius .15s ease, border .35s ease !default; // border transition matches speed of transition-collapse
```

Comment:
I think `border` transition here should be dropped or tweaked since `border-color` is already in transitioned in `$btn-transition` with `.15s`.

That may be causing the jump.

I tried a fw things without seeing anything so I'm not quite sure about this, but maybe using `$btn-transition` overlaps with this custom  `border` transition.

Reply:
well adding this was needed initially to fix the issue when it when the border was not transparent and going from a width of 1 to a width of 0. the issue may have been the fact that it's transitioning at `.15s`, but the height was transitioning at `.35s`, leading to one perceptible jump

Reply:
now that it's just a colour change that particular part is likely not causing a jump, but there's some fundamental other jump going on somewhere for the accordion as a whole now (i.e. it's not the panel content itself that's jumping visibly anymore, but everything else is...)